src/modules/rlm_attr_filter/rlm_attr_filter.c

```diff
@@ -133,8 +133,11 @@ static int attr_filter_getfile(TALLOC_CTX *ctx, rlm_attr_filter_t *inst, char co
 		     * then bitch about it, giving a good warning message.
 		     */
 		     if (fr_dict_attr_is_top_level(vp->da) && (vp->da->attr > 1000)) {
-			WARN("[%s]:%d Check item \"%s\"\n\tfound in filter list for realm \"%s\".\n",
-			       filename, entry->lineno, vp->da->name, entry->name);
+			 char *str;
+			 fr_value_box_aprint(NULL, &str, entry->name, NULL);
```

Comment:
You should be able to just pass in the `fr_value_box_t` directly, which is simpler:

```
 WARN("[%s]:%d Check item \"%s\"\n\tfound in filter list for realm \"%pV\".\n",
			      filename, entry->lineno, vp->da->name, entry->name);
```

Reply:
That gives a compiler error:
src/modules/rlm_attr_filter/rlm_attr_filter.c:27:20: warning: format ‘%s’ expects argument of type ‘char *’, but argument 10 has type ‘fr_value_box_t *’ {aka ‘struct value_box_s *’} [-Wformat=]

Reply:
Fixup the format arg where you're passing the value box to be a `%pV`